src/org/opendatakit/briefcase/ui/export/components/ConfigurationPanelForm.java

```diff
@@ -89,26 +91,14 @@
         buildPemFileDialog().choose().ifPresent(file -> setPemFile(Paths.get(file.toURI())))
     );
     pemFileClearButton.addActionListener(__ -> clearPemFile());
+
     startDatePicker.addDateChangeListener(event -> {
-      if (!isDateRangeValid()) {
-        showError(MessageStrings.INVALID_DATE_RANGE_MESSAGE, "Export configuration error");
-        startDatePicker.clear();
-      } else
-        onSelectStartDateCallbacks.forEach(consumer -> consumer.accept(extractDate(event)));
-    });
-    endDatePicker.addDateChangeListener(event -> {
-      if (!isDateRangeValid()) {
-        showError(MessageStrings.INVALID_DATE_RANGE_MESSAGE, "Export configuration error");
-        endDatePicker.clear();
-      } else
-        onSelectEndDateCallbacks.forEach(consumer -> consumer.accept(extractDate(event)));
+      endDatePicker.setEnabled(true);
+      endDatePicker.getSettings().setDateRangeLimits(extractDate(event), null);
+      setEndDate(extractDate(event));
```

Comment:
Updating the end date could pass unnoticed by the user and feels a little bit weird even when you are setting trying to set a date after the established end date

Reply:
Ok then I will clear it? Is that fine?

Reply:
Maybe the best option would be to clear the end date only if it's before the start date we're trying to set. What do you think?

Reply:
That situation wont be possible as such dates would be disabled already.

Reply:
Hmmm... If I set start to `2018-01-01`, the end date to `2018-01-20`, then I can set the start to `2018-01-30`, producing the situation I'm talking about. Then I would clear the end field.

If the second time I set the start date to `2018-01-10` instead, I wouldn't do anything, because the date range is valid.

Reply:
Oh okay sorry didn't think of this. Yes, this I will handle and clear the end date.